babel/util.py

```diff
@@ -255,7 +255,12 @@ def wraptext(text: str, width: int = 70, initial_indent: str = '', subsequent_in
 
 
 class FixedOffsetTimezone(datetime.tzinfo):
-    """Fixed offset in minutes east from UTC."""
+    """
+    Fixed offset in minutes east from UTC.
+
+    DEPRECATED: Use the standard library `datetime.timezone` instead.
```

Comment:
(Drive-by review, I'll try to review this thoroughly later) We could also add a module getter that raises a deprecation warning when this class is accessed/imported (or even when it's instantiated maybe?)

Reply:
Yeah, why not!